### PR TITLE
fix(rxAge): Fixes parseInt bug

### PR DESCRIPTION
**Pull request by lentusignavus**
JIRA: https://jira.rax.io/browse/FRMW-1000

### LGTMs
- [x] Dev LGTM
- [x] Dev+Vidyo LGTM
- [x] QE LGTM

With really small durations moment returns a small float
parseInt handles this weirdly returning 1 and sometimes 2
Changes implementation to use Math.floor

Issue: https://github.com/rackerlabs/encore-ui/issues/1767

### CODE REVIEW

File: src/utilities/rxAge/scripts/rxAge.js [hunk 1]
```diff
@@ -73,9 +73,9 @@ angular.module('encore.ui.utilities')
         var date = moment(new Date(dateString));
         var diff = now.diff(date);
         var duration = moment.duration(diff);
-        var days = parseInt(duration.asDays(), 10);
-        var hours = parseInt(duration.asHours(), 10);
-        var mins = parseInt(duration.asMinutes(), 10);
+        var days = Math.floor(duration.asDays(), 10);
```
Comment (lentusignavus) [inline src/utilities/rxAge/scripts/rxAge.js]: Removing the 10, next commit

Comment (jonkruse00) [inline src/utilities/rxAge/scripts/rxAge.js]: just saying from MDN : 
<blockquote>
<b>Always specify this parameter</b> to eliminate reader confusion and to guarantee predictable behavior. Different implementations produce different results when a radix is not specified, usually defaulting the value to 10.
</blockquote>

**Comment by jonkruse00** [comment]
If I can LGTM I do

**Comment by Droogans** [comment]
The commit message "Update rxAge.js" is not very informative.

Can you change it to be more explanatory? Also, following the commit message format guidelines is always a plus.

**Comment by CITguy** [comment]
LGTM

**Comment by CITguy** [comment]
Marking "DO NOT MERGE" until #1787 is ready for merge.

**Pull request merged**